.monorepolint.config.mjs

```diff
@@ -241,7 +242,7 @@ function getTsconfigOptions(baseTsconfigPath, opts) {
  *  legacy: boolean,
  *  esmOnly?: boolean,
  *  customTsconfigExcludes?: string[],
- *  tsVersion?: "^5.5.2"|"^4.9.5",
+ *  tsVersion?: typeof LATEST_TYPESCRIPT_DEP | "^4.9.5",
```

Comment:
Just preventing future annoying behavior. Typescript doesn't actually check the .mjs files so everything was working here with any string but we were getting red in the IDE. This will at least prevent that mismatch.